runelite-client/src/main/java/net/runelite/client/plugins/woodcutting/WoodcuttingPlugin.java

```diff
@@ -190,11 +205,31 @@ public void onChatMessage(ChatMessage event)
 
 			if (event.getMessage().contains("A bird's nest falls out of the tree") && config.showNestNotification())
 			{
-				notifier.notify("A bird nest has spawned!");
+				if (clueTierSpawned == null || clueTierSpawned.getTier() >= config.clueNestNotifyTier().getTier())
+				{
+					notifier.notify("A bird nest has spawned!");
+				}
+				// Clear the clue tier that has previously spawned
+				clueTierSpawned = null;
 			}
 		}
 	}
 
+	@Subscribe
+	public void onItemSpawned(ItemSpawned itemSpawned)
+	{
+		if (CLUE_NEST_IDS.contains(itemSpawned.getItem().getId()))
+		{
+			// It seems that falling nests spawn before the chat message is displayed, thus we can save state here and notify on game message.
+			clueTierSpawned = clueNestIdToTier.get(itemSpawned.getItem().getId());
+		}
+		else if (NON_CLUE_NEST_IDS.contains(itemSpawned.getItem().getId()))
```

Comment:
This yields the same behavior?
```suggestion
		else
```

Reply:
Yeah it does, but consider this (very unlikely) case where some random item spawns the same tick as a clue bird nest. We might get `onItemSpawned` invoked for this random item before the chat message will be processed by `onChatMessage`, changing `clueTierSpawned` to `null` by this change which might cause us to notify on a too-low tier clue nest. I think we rather only set it to `null` upon seeing an actual non-clue nest.